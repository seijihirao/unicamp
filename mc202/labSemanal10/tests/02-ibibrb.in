i 323712940 -1
i 3676369188 -2
b 323712940
b 3676369188
i 323712940 -10
i 3676369188 20
b 323712940
b 3676369188
r 323712940
r 3676369188
b 323712940
b 3676369188
i 323712940 100
i 3676369188 -200
b 323712940
b 3676369188
r 323712940
r 3676369188
b 323712940
b 3676369188
f
