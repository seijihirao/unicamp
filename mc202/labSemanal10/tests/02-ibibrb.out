valor para 323712940: -1
valor para 3676369188: -2
valor para 323712940: -10
valor para 3676369188: 20
323712940 nao existe
3676369188 nao existe
valor para 323712940: 100
valor para 3676369188: -200
323712940 nao existe
3676369188 nao existe
